src/interchange/src/protobuf.rs

```diff
@@ -137,9 +147,9 @@ impl Decoder {
                     );
                 }
             } else {
-                let compiled = compile_proto(schema_id, client).await?;
+                let compiled = compile_proto_from_registry(schema_id, client).await?;
                 let schema_to_compare = DecodedDescriptors::from_bytes(
-                    &strconv::parse_bytes(&compiled.schema)?,
+                    &compiled.file_descriptor_set,
```

Comment:
oh I guess here it's different -- but why do we not pass in `compiled.message_name` and instead use `self.descriptors.message_name`?